src/main/java/org/opensearch/eval/samplers/ProbabilityProportionalToSizeParameters.java

```diff
@@ -10,7 +10,8 @@
 
 public class ProbabilityProportionalToSizeParameters extends AbstractSamplerParameters {
 
-    public ProbabilityProportionalToSizeParameters(final String name, final String description, final String sampling, final int querySetSize) {
+    public ProbabilityProportionalToSizeParameters(final String name, final String description, final String sampling,
```

Comment:
I think there's no need for `application` and `searchConfig` in the parameters of PPTSS if we don't have it in the "all" sampler?
Only when running a query set, not when creating a query set, the parameters are necessary.

Reply:
Dang it. Got 'em.